rts/Lua/LuaOpenGL.cpp

```diff
@@ -4724,11 +4724,28 @@ int LuaOpenGL::MultiTexGen(lua_State* L)
 /***
  * @function gl.BindImageTexture
  * @param unit integer
- * @param texID string?
- * @param level integer?
- * @param layer integer?
- * @param access integer?
- * @param format integer?
+ * @param texID nil | string (nil breaks any existing binding to the image unit)
```

Comment:
`string?` is the same as `nil | string`, I prefer the short form but it does not matter in the end.